new/LIT7038Qene.xml

```diff
@@ -5,7 +5,10 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
     <teiHeader>
         <fileDesc>
             <titleStmt>
-                <title xml:lang="en" xml:id="t1">Qǝne poem on ʾEwosṭātewos</title>
+                <title xml:lang="gez" xml:id="t1">እስመ፡ ወርኃ፡ ተድላ፡ ኮኖሙ፡ ለአዕባነ፡ ፍኖት፡ አፈ፡</title>
+                <title xml:lang="gez" corresp="#t1" type="normalized">ʾƎsma warḫā tadlā konomu la-ʾaʿǝbāna fǝnot ʾaf</title>
+                <title xml:lang="en" corresp="#t1">When its month is delight, there was a brim of the stones of the way</title>
```

Comment:
Please add ... in transcription (followed by the indication Qǝne of ... type) and and transliteration, to show that the title is quotation.